apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/Aggregations.cs

```diff
@@ -32,4 +32,56 @@ public static class Aggregations
     /// <returns>A count(*) aggregation.</returns>
     public static Aggregation Count(string alias = "") =>
         new Aggregation { Count = new Count(), Alias = alias };
+
+    /// <summary>
+    /// Sum of the values of the requested property.
```

Comment:
Please use XML documentation formatting instead of Markdown for lists and paragraphs. It will show up better in VS.

See https://github.com/googleapis/google-cloud-dotnet/blob/main/apis/Google.LongRunning/Google.LongRunning/Operation.cs#L25 for an example.

If there's a Datastore product documentation page with all this information, I'd actually prefer a link to that, so that we don't need to worry about keeping this up to date.

(This all applies to Sum as well.)

Reply:
+1 for waiting for Datastore to have public documentation around Avg and Sum and link that here. This is the kind of documentation that tends to get out of date really quickly.